### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by martinconstante**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+-   Reduce the total cost of ownership for homegrown technology supporting their indirect channel
+-   Standardize integrations with partners
+-   Increase efficiencies and minimize redundancies by bridging their direct and indirect sales channels
+
+Distributors can use CloudBlue Connect to:
+-   Define, manage and distribute any type of product (omni-product) through any channel (omni-channel)
+-   Transform perpetual licensed products into a subscription model
+-   Onboard new products into their portfolio quickly to build and deliver unique solutions to end customers
+
+Mulesoft Connector for Cloudblue Connect enables you to create apps that react to Cloudblue Connect events such as adding, changing, or deleting objects like purchase requests, assets, accounts.
```
Comment (marcserrat) [inline README.md:16]: Connector => extension

File: README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Cloudblue Connect Mulesoft Extension
+
+## Overview
+CloudBlue Connect is a supply automation platform that manages your products and services, contracts, ordering and fulfillment, usage and subscriptions.
```
Comment (marcserrat) [inline README.md:4]: CloudBlue Connect is a supply automation platform that between others manages products, services, contracts, ordering and fulfillment, usage and subscriptions.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+-   Increase efficiencies and minimize redundancies by bridging their direct and indirect sales channels
+
+Distributors can use CloudBlue Connect to:
+-   Define, manage and distribute any type of product (omni-product) through any channel (omni-channel)
+-   Transform perpetual licensed products into a subscription model
+-   Onboard new products into their portfolio quickly to build and deliver unique solutions to end customers
+
+Mulesoft Connector for Cloudblue Connect enables you to create apps that react to Cloudblue Connect events such as adding, changing, or deleting objects like purchase requests, assets, accounts.
+
+It allows an easy and fast integration with the Cloud Blue Connect Platform. Thanks to it you can automate the fulfillment of orders generated by your products.
```
Comment (marcserrat) [inline README.md:18]: Cloud Blue => CloudBlue

File: README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Cloudblue Connect Mulesoft Extension exposes methods for accessing Cloudblue Connect and uses the API defined in [Cloudblue Connect OpenAPI Specification](https://connect.cloudblue.com/community/api/openapi/).
```
Comment (marcserrat) [inline README.md:20]: uses the API defined in => uses the public API defined at

File: README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+
+This extension has all of the functionallity exposed in the API definition, and provides you with the lowest-level access to the functionality of the Connect Platform.
```
Comment (marcserrat) [inline README.md:23]: the lowest-level access to => all access to

File: README.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+## Prerequisites
+To use this connector, you must be familiar with:
+
+-   The Cloudblue Connect operation    
+-   Anypoint Connectors
```
Comment (marcserrat) [inline README.md:29]: Anypoint Studio

File: README.md (hunk 1 above)
Comment (martinconstante) [reply README.md:16]: done

File: README.md (hunk 2 above)
Comment (martinconstante) [reply README.md:4]: Done

File: README.md (hunk 3 above)
Comment (martinconstante) [reply README.md:18]: Done

File: README.md (hunk 4 above)
Comment (martinconstante) [reply README.md:20]: Done

File: README.md (hunk 5 above)
Comment (martinconstante) [reply README.md:23]: Done

File: README.md (hunk 6 above)
Comment (martinconstante) [reply README.md:29]: Done

**Review by marcserrat — APPROVED** [review]

File: README.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+Anypoint Studio Allows to build, test and deploy Mulesoft Apps in the Mulesoft Cloud Platform or in on-prem platform. 
+The documentation to deploy the connector in Anypoint studio is available [here](https://connect.cloudblue.com/community/extensions/mulesoft/cloudblue-connect-mulesoft-extension/install-connector-to-use-in-anypoint-studio/).
+
+### Install Connector to use in Design Center
+Once you deploy the connector in Anypoint Exchange Platform, you can share, download or use in the Design Center Platform.
+The documentation to deploy the connector in Anypoint Exchange Platform is available [here](https://connect.cloudblue.com/community/extensions/mulesoft/cloudblue-connect-mulesoft-extension/install-connector-to-use-in-design-center/).
+
+## Build an Mulesoft app using Cloudblue Connect Mulesoft Extension
+In Mulesoft ecosistem, an integration between two or more platforms is called Application or App.
+It's possible to build an App in both platforms using the cloudblue extension.
```
Comment (rahulmondal) [inline README.md:49]: If you have any documentation page with example app, then linking here will be great.

File: README.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+-   The Cloudblue Connect operation    
+-   Anypoint Studio    
+-   Mule runtime engine (Mule)   
+-   Elements and global elements in a Mule flow    
+-   Creating a Mule app using Anypoint Studio (Studio)
+
+## Getting Started
+
+In this repository you will find the code to deploy the Cloudblue Connect Mulesoft Extension. Also you can download from Maven repository.
```
Comment (rahulmondal) [inline README.md:36]: The code in this repo is not recommended for self build and deploy. Better to guide the viewer to only Maven Repository.
Also include dependency code sample to include in pom.xml

**Review by rahulmondal — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (martinconstante) [reply README.md:49]: No yet.

File: README.md (hunk 8 above)
Comment (martinconstante) [reply README.md:36]: If you want to install in Anypoint Exchange you must download the repo, modify the pom.xml with the specific data from the user and deploy... then both options are possibles.

**Review by rahulmondal — APPROVED** [review]

**Pull request merged**